### PR TITLE
Kickoff script

### PR DESCRIPTION
**Pull request by mishuagopian**
Referencing issue: https://github.com/Wolox/frontend-bootstrap/issues/60.

The script does the following:
> Changes projects data in package.json and bower.json
> Change project name and details in README.md. Also removes license and contributing.

### CODE REVIEW

File: script/bootstrap.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	bfm = require('./bootstrapFileManager');
+
+prompt.colors = false;
+prompt.message = '';
+prompt.delimiter = '';
+prompt.start();
+
+var config = {
+	properties: {
+		responsibleUername: {
```
Comment (sbalay) [inline script/bootstrap.js]: responsibleU**s**ername

**Comment by sbalay** [comment]
Cool!
thanks

**Comment by sbalay** [comment]
Thinking about this? Do you think that this script should also remove the ```scripts``` folder?

**Comment by mishuagopian** [comment]
Sounds good. I'll try to do it but I'm not sure if I can.

**Comment by mishuagopian** [comment]
@sbalay Everything fixed!

**Pull request merged**